packages/governance/package.json

```
@@ -53,7 +53,8 @@
     "@endo/bundle-source": "^3.5.0",
     "@endo/init": "^1.1.7",
     "ava": "^5.3.0",
-    "c8": "^10.1.2"
+    "c8": "^10.1.2",
+    "@endo/compartment-mapper": "^1.4.0"
```

This is now needed in packages/internal/package.json rather than here.
```suggestion
    "c8": "^10.1.2"
```